client/src/components/Menu/index.tsx

```diff
@@ -107,6 +113,8 @@ class Menu extends React.Component<IMenuProps> {
     public render(): JSX.Element {
         const {loggedUsername, newUsername, searchResults, loaded}: IMenuProps = this.props;
         const {searchValue}: IMenuState = this.state;
+        const sharp = 'e';
```

Comment:
why e?

Reply:
I tested with this variable e, i changed on #